### PR TITLE
Legal

### PR DESCRIPTION
**Pull request by klucar**
New PR for updating legal bits. I did each one in a separate commit. So far it contains:

LICENSE/NOTICE update
Typo-fix
Apache Headers on all java files.

I'll update with namespace change after other outstanding PRs are merged.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,55 +1,213 @@
...
+   limitations under the License.
+
+
+======================================================================
+Apache Myriad Subcomponents:
+
+The Apache Myriad project contains subcomponents with separate copyright
+notices and license terms. Your use of the source code for the these
+subcomponents is subject to the terms and conditions of the following
+licenses.
```
Comment (adam-mesos) [inline LICENSE]: What subcomponents, and what following licenses? Are we adding these in a later commit?

Comment (klucar) [inline LICENSE]: I created a JIRA to discuss this. Short version, our NOTICE file had some
outdated dependencies listed.

https://issues.apache.org/jira/browse/MYRIAD-150

**Comment by kensipe** [comment]
@klucar is there an open discussion at that JIRA prior to merging this?  or is this ready for more review in order merge?

**Comment by klucar** [comment]
I had created it as a reminder, then Adam brought it up so I linked the
JIRA. I don't think there has been any discussion so far other than the old
one was out of date.

**Comment by kensipe** [comment]
LGTM  any reason to hold up?  I'm ready to merge.

**Comment by kensipe** [comment]
@klucar myriad-scheduler/src/main/resources/webapp/public/js/myriad.js
has a collision, it shows in your commit 1000+ additions and 700+ deletions... what did you do to this file?

**Comment by klucar** [comment]
I added the apache header to the index.html file and re-generated the site.
everything in public is auto-generated.

**Comment by kensipe** [comment]
merged!

**Pull request merged**